applications/starlay_finance.md

```diff
@@ -0,0 +1,143 @@
+# Starlay Finance
+
+- **Team Name:** Starlay Finance
+- **Payment Address:** `5EeWRBfmWJo1ZnAbCWSupbA4KEiyPnYBJZKid9ngFcheH94g`
+
+## Project Overview :page_facing_up:
+### Overview
+
+This project demonstrates the Starlay team's dedication to advancing the DeFi space by developing a WebAssembly (Wasm) version of Compound V2, which bolsters both its performance and security capabilities. By making the project's source code open and accessible, we aim to empower developers within the Polkadot ecosystem to seamlessly and securely integrate lending-related functionalities using Wasm technology. This Wasm lending contract is designed to be compatible with all Wasm-supporting parachains, including prominent networks such as Astar Network, Phala Network, Alphazero, and more.　
```

Comment:
Will this implementation differ from Compound V2 in any way, or will it basically just be an ink! port?

Reply:
Not differ from Compound V2. This implementation will basically just be an ink! port.